components/application-mgt/org.wso2.carbon.identity.application.mgt.ui/src/main/java/org/wso2/carbon/identity/application/mgt/ui/ApplicationBean.java

```diff
@@ -1074,6 +1079,44 @@ public void update(HttpServletRequest request) {
         serviceProvider.setApplicationName(request.getParameter("spName"));
         serviceProvider.setDescription(request.getParameter("sp-description"));
         serviceProvider.setCertificateContent(request.getParameter("sp-certificate"));
+        String jwks = request.getParameter(ApplicationMgtUIConstants.JWKS_URI);
+        boolean jwksExist = false;
+        ArrayList<ServiceProviderProperty> spPropList;
+        //adding jwks uri as a sp meta data
```

Comment:
It is better to have inline comments in the following format. Always start with a capital letter and end with a full stop. 
// \<Comment\>.

Reply:
Please fix all the places to retain consistancy.